_posts/0000-01-02-TenronOtrin.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Do I know what I'm doing? Perhaps not, which is why I'm following this tutorial!